inventory/s3/orc_utils.go

```diff
@@ -0,0 +1,125 @@
+package s3
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go/service/s3/s3iface"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	gproto "github.com/golang/protobuf/proto" //nolint:staticcheck // orc lib uses old proto
+	"github.com/scritchley/orc/proto"
+	"github.com/treeverse/lakefs/logging"
+	"modernc.org/mathutil"
+)
+
+const (
+	maxPostScriptSize  = 256
```

Comment:
Is this somehow part of the ORC standard or guaranteed by AWS?  If so, please link to docs (it may change).  Otherwise, why 256?